pkg/sql/vecindex/mutation_searcher.go

```diff
@@ -30,17 +31,21 @@ type MutationSearcher struct {
 	idxCtx       cspann.Context
 	partitionKey tree.Datum
 	encoded      tree.Datum
+	evalCtx      *eval.Context
 }
 
 // Init wraps the given KV transaction in a C-SPANN transaction and initializes
 // the search context.
 //
 // NOTE: The index is expected to come from a call to Manager.Get, and therefore
 // using a vecstore.Store instance.
-func (s *MutationSearcher) Init(idx *cspann.Index, txn *kv.Txn, tableDesc catalog.TableDescriptor) {
+func (s *MutationSearcher) Init(
```

Comment:
NIT: usually we put `evalCtx` as the first param (or 2nd after `context.Context`).